app/views/bus_stops/outdated.haml

```diff
@@ -0,0 +1,26 @@
+Bus stops not updated since #{@date} (#{@stops.count})
+
+= form_tag '#', method: :get do
+  = text_field_tag :date, @date, class: 'datepick'
+  = submit_tag 'Change date'
+
+= link_to 'Export as CSV', outdated_bus_stops_path(date: params[:date], format: :csv)
```

Comment:
I tried `..._path(params, format: :csv)` for future proofing sake, but that didn't work.